src/content/news/2025-03-03-maplibre-newsletter-february-2025/index.mdx

```diff
@@ -0,0 +1,210 @@
+  />
+  <div
+    style={{
+      fontSize: "0.9em",
+      color: "#666",
+      marginTop: "8px",
+      textAlign: "center",
+    }}
+  >
+    Screenshot showing the road network in Europe from some demo MLTs based on
+    Bing Maps data.
+  </div>
+</div>
+
+- [MapLibre iOS 6.11.0](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.11.0) is now available.
+
+  - Custom Style Layers, allow drawing directly on the map using the Metal API with full Swift compatibility. Learn more in [this guide](https://maplibre.org/maplibre-native/ios/latest/documentation/maplibre-native-for-ios/customstylelayerexample).
+
+- Looking ahead, the upcoming MapLibre iOS 6.12.0 release will expose some low-level events for better observability. More details can be found in [this article](https://maplibre.org/maplibre-native/ios/latest/documentation/maplibre-native-for-ios/observerexample).
+- Custom Drawable Layers allow drawing on the map in a platform-independent way. Currently C++ is required to implement these layers. This month [a significant PR](https://github.com/maplibre/maplibre-native/pull/3210) was merged, introducing several imporvements, including:
+
+  - enhanced functionality and improvements to custom drawable layer,
+  - support for uploading 3D geometries, expanding rendering possibilities.
+
+  {" "}
+
+  <div
+    style={{
+      display: "flex",
+      justifyContent: "center",
+      flexDirection: "column",
+      alignItems: "center",
+    }}
+  >
+    <Image
+      src={custom_draw_layer_demo}
+      style={{
+        maxWidth: "550px",
+        width: "100%",
+        height: "auto",
+        display: "inline-block",
+      }}
+      alt="Custom Drawable Layer demo available in the GLFW app if you press 'V'."
+    />
+    <div
+      style={{
+        fontSize: "0.9em",
+        color: "#666",
+        marginTop: "8px",
+        textAlign: "center",
+      }}
+    >
+      Custom Drawable Layer demo available in the GLFW app if you press 'V'.
+    </div>
+  </div>
+
+- [@frankkienl](https://github.com/frankkienl) from Flitsmeister has created a sample app demonstrating how to integrate MapLibre with Android Auto (and Automotive). This is a fantastic starting point for developers looking to explore automotive applications. Check out the repository here: [MapLibre Android Auto Sample](https://github.com/maplibre/MapLibre-Android-Auto-Sample).
+
+## Web
+
+February was a shorter month, but we still managed to roll out [MapLibre GL JS 5.1.1](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.1.1) with key improvemens.
+
+One notable highlight is the powerful `addProtocol` feature which enabled the release of a new plugin for loading arbitrary fonts using tinySDF:Check out [maplibre-local-glyphs](https://www.npmjs.com/package/maplibre-local-glyphs), developed by [@spatialillusions](https://github.com/spatialillusions) - a big thanks to their contribution!
```

Comment:
Maybe I am reading it wrong but this makes  it sound `addProtocol` is a new feature. I would consider adjusting the wording a bit. E.g.:

The [`addProtocol` feature](https://maplibre.org/maplibre-gl-js/docs/API/functions/addProtocol/) is now even more powerful, with support for...

Reply:
modified, please see if it sounds right.

Reply:
I'm good with the current phrasing, `addProtocol` is as powerfull as it was before this plugin and will remain that way probably :-)